packages/core/src/bootstrap.ts

```diff
@@ -218,84 +218,62 @@ const messageReceivedHandler = async ({
         shouldRespondPrompt
       );
 
-      const response = await runtime.useModel(ModelType.TEXT_SMALL, {
-        prompt: shouldRespondPrompt,
-      });
+      let responseContent: Content | null = null;
+      // Retry if missing required fields
+      let retries = 0;
+      const maxRetries = 3;
```

Comment:
Should we make it configurable?

Reply:
maybe, but there are so many retries through the system, I don't want to flood with ENV vars. Are there any existing patterns for this type of tunable?